### ISSUE TITLE
invalid type field for default precision statement

### ISSUE THREAD

**Issue by MikuAuahDark** [love2d/love-android#204]
Someone in Discord reported crash when trying to run LOVE in Android with this device information:
* Device: Samsung Galaxy J1 (SM-J100FN)
* Android: 4.4.4

Error traceback as follows:
```
boot.lua:535: Cannot compile vertex shader code:
(74:0) : error : invalid type field for default precision statement
(75:0) : error : invalid type field for default precision statement

Traceback
[C]: in function 'setMode'
[C]: in function 'xpcall'
[C]: in function 'xpcall'
```

Also there's same issue on forums too: https://love2d.org/forums/viewtopic.php?t=88677

**Comment by MikuAuahDark** [comment]
[app-normal-release-unsigned.apk.zip](https://github.com/love2d/love-android/files/6330430/app-normal-release-unsigned.apk.zip)

That APK contains fixes from love2d/love@e9c58c8192ff08f30539e6a0d80e4b5f3abaa3ce. Can't upload it to Discord due to 8MB limit.

**Comment by MikuAuahDark** [comment]
We'll assume it's fixed.
